cli/pkg/kctrl/cmd/app/release/release_cmd_runner_test.go

```diff
@@ -0,0 +1,21 @@
+package release
+
+import (
+	"bytes"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestReleaseCmdRunnerForKappCmd(t *testing.T) {
+	var buf bytes.Buffer
+	ReleaseCmdRunner := NewReleaseCmdRunner(&buf, false, "", false, nil)
+	cmd := exec.Command("kapp", "deploy", "-f", "-", "-a", "pkg-test", "-y")
+	err := ReleaseCmdRunner.RunWithCancel(cmd, nil)
+	require.NoError(t, err)
+	expectedLength := 0
+	if actualLength := buf.Len(); actualLength != expectedLength {
+		t.Errorf("Got Buffer length = %d, Expected empty", actualLength)
```

Comment:
Let's use require here, and should we point kapp to an actual file?

Reply:
Done: https://github.com/carvel-dev/kapp-controller/pull/1638/commits/6cb1162994523fdfa9a1ee8bcaef50fd103784b3